editor/editor_help.cpp

```diff
@@ -888,10 +921,26 @@ void EditorHelp::_update_doc() {
 			class_desc->append_text(TTR("There is currently no description for this class. Please help us by [color=$color][url=$url]contributing one[/url][/color]!").replace("$url", CONTRIBUTE_URL).replace("$color", link_color_text));
 		}
 
+		class_desc->add_newline();
+		class_desc->add_newline();
+	}
+
+#ifdef MODULE_MONO_ENABLED
+	if (classes_with_csharp_differences.has(cd.name)) {
+		const String &csharp_differences_url = vformat("%s/tutorials/scripting/c_sharp/c_sharp_differences.html", VERSION_DOCS_URL);
+
+		class_desc->push_color(theme_cache.text_color);
+		_push_normal_font();
+		class_desc->push_indent(1);
+		_add_text("[b]" + TTR("Note:") + "[/b] " + TTR("There are notable differences when using this API with C#. See [url=" + csharp_differences_url + "]C# API differences to GDScript[/url] for more information."));
```

Comment:
`TTR` can't be a concatenated string, it needs to be a literal that can be extracted by a python script. So you should use `vformat()` to compose the string.

Reply:
I noticed other `TTR` strings seem to use `$url` and `.replace("$url", CONTRIBUTE_URL)`.